src/common/style.js

```diff
@@ -61,7 +61,7 @@ export const mix = (...styles/*:Array<?Rules>*/)/*:Rules*/ => {
   }
 
   const composedStyle = id !== null ?
-    composedStyles[id/*::.toString()*/] :
+    composedStyles[id.toString()] :
```

Comment:
This was actually commented out for a reason, should have added comment explaining. `id` in this case is a `symbol` but flow does not yet support computed property access via non string `id` so this is a hack to make flow thing we're looking up via string rather than symbol.